packages/twenty-server/src/engine/metadata-modules/field-metadata/composite-types/domain.composite-type.ts

```diff
@@ -0,0 +1,27 @@
+import { CompositeType } from 'src/engine/metadata-modules/field-metadata/interfaces/composite-type.interface';
+
+import { FieldMetadataType } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+
+export const domainCompositeType: CompositeType = {
+  type: FieldMetadataType.DOMAIN,
+  properties: [
+    {
+      name: 'primaryLink',
+      type: FieldMetadataType.TEXT,
+      hidden: false,
+      isRequired: true,
+    },
+    {
+      name: 'secondaryLinks',
+      type: FieldMetadataType.TEXT,
```

Comment:
should be an array?

Reply:
As discussed with @charlesBochet, now this property is now a RAW_JSON.